src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationFeedbackPresenter.cs

```diff
@@ -25,13 +25,12 @@ public interface IParameterIdentificationRunFeedbackPresenter : IParameterIdenti
 
    public interface IParameterIdentificationFeedbackPresenter : IPresenter<IParameterIdentificationFeedbackView>,
       IToogleablePresenter,
-      IListener<ParameterIdentificationStartedEvent>,
-      IListener<ParameterIdentificationSelectedEvent>,
-      IListener<ParameterIdentificationTerminatedEvent>,
       IListener<ParameterIdentificationIntermediateResultsUpdatedEvent>,
       IListener<ProjectClosedEvent>
    {
       bool ShouldRefreshFeedback { get; set; }
+      void OnParameterIdentificationStarted(ParameterIdentification parameterIdentification);
```

Comment:
OnXxx looks like an event handler but it not. Why not just call it ParmaeterIdentificationStarted?